pallet-logion-loc/src/lib.rs

```diff
@@ -86,14 +86,28 @@ impl<AccountId, LocId, EthereumAddress> Default for Requester<AccountId, LocId,
     }
 }
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, TypeInfo, Copy)]
+pub enum SupportedAccountId<AccountId, EthereumAddress> {
+    None, // Enables "null" account ID
+    Polkadot(AccountId),
+    Ethereum(EthereumAddress)
+}
```

Comment:
I would reuse existing naming and type:
```suggestion
pub enum SupportedAccountId<AccountId, EthereumAddress> {
    Account(AccountId),
    OtherAccount(OtherAccountId<EthereumAddress>),
}
```
which brings 2 benefits:
* Easier to read/maintain code,
* Extending the type `OtherAccountId` (e.g. to support Bitcoin addresses) will automatically enable new submitters.

Reply:
Agreed.